circle.yml

```diff
@@ -10,3 +10,11 @@ test:
   override:
     - docker run civisanalytics/civis-jupyter-python3 /bin/bash -c "echo BUILDS OK"
     - docker run civisanalytics/civis-jupyter-python3 python -c "import civis"
+
+deployment:
+  release:
+    tag: /v[0-9]+(\.[0-9]+)*/
+    commands:
+      - go get github.com/tcnksm/ghr
```

Comment:
I think all that needs to be done is get rid of the 2 lines with ghr and add a line that uploads the requirements.txt using the AWSCLI. See: https://circleci.com/docs/1.0/continuous-deployment-with-amazon-s3/

Reply:
👍